### ISSUE TITLE
Remove use of extract()

### ISSUE THREAD

**Issue by Wiljami** [Seravo/wp-custom-bulk-actions#7]
The plugin makes use of extract() function which is problematic as it hides where variables are coming from.

WordPress has it deprecated and is working to remove its use: https://core.trac.wordpress.org/ticket/22400

Should this plugin be rewritten to function without it?

**Comment by ottok** [comment]
If the plugin has very few active installations and would require a
lot of maintenance effort to keep on wp.org, then we can always just
choose to explicitly stop maintaining it and remove it from wp.org.

**Comment by Wiljami** [comment]
I agree. From the looks of this project, it never reached a very mature state and hasn't been kept up to date. 

Removing plugins that are out of date and unsupported from wp.org would be a good idea.
